tutorials/convert_lit_models.md

```diff
@@ -115,5 +115,36 @@ import torch
 from transformers import AutoModel
 
 state_dict = torch.load('out/hf-tinyllama/converted/model.pth')
-model = AutoModel.from_pretrained("TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T", state_dict=state_dict)
+model = AutoModel.from_pretrained("checkpoints/TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T", state_dict=state_dict)
 ```
+
+&nbsp;
+## Using the LM Evaluation Harness
+
+To evaluate LitGPT models, use the integrated evaluation utilities based on Eleuther AI's LM Evaluation Harness. For more information, please see the [evaluation](evaluation.md) documentation.
+
+Alternatively, if you wish to use converted LitGPT models with the LM Evaluation Harness from [Eleuther AI's GitHub repository](https://github.com/EleutherAI/lm-evaluation-harness), you can use the following steps.
+
+1. Follow the instructions above to load the model into a Hugging Face transformers model.
+
+2. Create a `model.safetensor` file:
+
+```python
+model.save_pretrained("out/hf-tinyllama/converted/")
+```
+
+3. Copy the tokenizer files into the model-containing directory:
```

Comment:
Hmm, I was missing this step. This probably explains why I was never successful just converting the checkpoint only (and getting cryptic error). Dang

Reply:
Hm, yeah, I think the HF files needed contain more than just the state dict.